lib/protobuf3_fixer/encoder.rb

```diff
@@ -18,7 +18,7 @@ def reencoded_hash
     def clean_encoded_json_for_klass
       case required_typefix
       when :timestamp
-        Time.at(data['seconds'], (data['nanos'] || 0) / 10**6).utc.to_datetime.rfc3339
+        DateTime.rfc3339(data).to_time.utc.to_datetime.rfc3339
```

Comment:
This shouldn't be required if they fixed the Timestamp parsing?

Reply:
without that we get this in the specs:

```
       -"bar" => {"a"=>"1970-01-01T00:00:10+00:00"},
       -"foo" => ["1970-01-01T00:00:10+00:00"],
       -"ts" => "1970-01-01T00:00:10+00:00",
       -"tsOne" => "1970-01-01T00:00:10+00:00",
       -"tsTwo" => "1970-01-01T00:00:10+00:00",
       +"bar" => {"a"=>"1970-01-01T00:00:10Z"},
       +"foo" => ["1970-01-01T00:00:10Z"],
       +"ts" => "1970-01-01T00:00:10Z",
       +"tsOne" => "1970-01-01T00:00:10Z",
       +"tsTwo" => "1970-01-01T00:00:10Z",
```

Reply:
cool! one more thing we don't have to hack out outselves